CHANGELOG.md

```diff
@@ -95,7 +96,7 @@ You can find all deprecations in [this overview](https://github.com/kedacore/ked
 
 New deprecation(s):
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- Various Prometheus metrics have been renamed to follow the preferred naming conventions. The old ones are still available, but will be removed in the future ([#4854](https://github.com/kedacore/keda/issues/4854)).
```

Comment:
@JorTurFer, @zroubalik, @tomkerkhove I am not entirely sure this is the right place to document the deprecation, I took it from the original PR https://github.com/kedacore/keda/pull/5174.

Reply:
We'll need a new issue for this; if I understand correctly we are just renaming them and keep the old ones. Once our doc PR is ready I will create required issue/discussions